<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Request</class>
 <widget class="QDialog" name="Request">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="YesButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>93</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NoButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>93</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>258</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="PlayerInformation">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>拒绝理由</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RejectEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>YesButton</sender>
   <signal>clicked()</signal>
   <receiver>Request</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NoButton</sender>
   <signal>clicked()</signal>
   <receiver>Request</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
